DEBUGGING.md

```diff
@@ -0,0 +1,51 @@
+# ember-m3
+
+## Debugging
+
+If you are used to interacting with `DS.Model` records, it can be intimidating to see
+the `m3` proxy object in the console:
```

Comment:
```suggestion
a proxy object wrapping an `m3` model in the console:
```

I realize this is pretty pedantic and TBH I don't mind what you've written, but strictly speaking the proxy object you see there doesn't come from m3 (you can observe by looking at such a record in production, where it will not be wrapped in this proxy).